buildSrc/src/main/kotlin/com/datadog/gradle/config/MavenConfig.kt

```diff
@@ -107,7 +107,12 @@ fun Project.publishingConfig(projectDescription: String) {
         signingExtension.apply {
             val privateKey = System.getenv("GPG_PRIVATE_KEY")
             val password = System.getenv("GPG_PASSWORD")
-            isRequired = true
+            setRequired {
```

Comment:
Although this does work, I'd rather we had a better solution, e.g. set an IS_CI environment variable (if none already exist), and use that to define if the signing is required or not.

Reply:
I would personally avoid relying on env variable in that flow, because unlike in the case of missing credentials, when publication will fail, in case of misconfigured environment publication will still succeed (without signing) and may stay unnoticed for a long time (because normally CI logs are not checked each time). That is why I wanted to keep it always `true` and disable only if explicitly requested by the user.

Maybe we can introduce a project property then? which should be added to the `gradlew` execution to disable signing? and if property is missing, then it will be `true`. wdyt?

Reply:
went for a property `dd-skip-signing`: if it is present, then signing will be skipped, otherwise it is effective.